### PR TITLE
API example notebook improvements

### PR DESCRIPTION
**Pull request by PeterLombaers**
The example simulation now uses a fixed seed. There is an example of getting the settings from the state. There is an example of joining the state to the dataset.

### CODE REVIEW

File: docs/source/example_api_asreview_file.ipynb [hunk 1]
```diff
@@ -629,6 +1097,31 @@
...
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'settings': {'model': 'nb', 'query_strategy': 'max', 'balance_strategy': 'double', 'feature_extraction': 'tfidf', 'n_instances': 1, 'stop_if': 'min', 'n_prior_included': 1, 'n_prior_excluded': 1, 'model_param': {'alpha': 3.822}, 'query_param': {}, 'feature_param': {'ngram_max': 1, 'stop_words': 'english', 'split_ta': 0, 'use_keywords': 0}, 'balance_param': {'a': 2.155, 'alpha': 0.94, 'b': 0.789, 'beta': 1.0}}, 'state_version': '1', 'software_version': '1.0+8.gf67a5bb', 'model_has_trained': True}\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open_state('example.asreview') as state:\n",
+    "    print(state.settings_metadata)"
```
Comment (J535D165) [inline docs/source/example_api_asreview_file.ipynb:1122]: Maybe use `pprint`?

File: docs/source/example_api_asreview_file.ipynb [hunk 2]
```diff
@@ -36,7 +36,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
```
Comment (J535D165) [inline docs/source/example_api_asreview_file.ipynb:39]: Please reset the kernel and rerun the notebook. This avoids non-linear execution errors and changes in git.

**Review by J535D165 — CHANGES_REQUESTED** [review]
Thanks!

File: docs/source/example_api_asreview_file.ipynb [hunk 3]
```diff
@@ -85,18 +85,18 @@
     {
      "data": {
       "text/plain": [
-       "{'version': '1.0rc1+46.gb310417.dirty',\n",
+       "{'version': '1.0+8.gf67a5bb',\n",
```
Comment (Rensvandeschoot) [inline docs/source/example_api_asreview_file.ipynb:88]: Should we change this to v1.1 when we release the next version of ASReview LAB?

File: docs/source/example_api_asreview_file.ipynb [hunk 4]
```diff
@@ -472,27 +489,478 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "There are also multiple functions to get one specific type of data directly. For example, to obtain the labeling times and plot them in a graph, use the following code: "
+    "You can join the data in the state to the original dataset."
```
Comment (Rensvandeschoot) [inline docs/source/example_api_asreview_file.ipynb:492]:
```suggestion
    "You can merge the information from the state file with the original dataset."
```

File: docs/source/example_api_asreview_file.ipynb [hunk 5]
```diff
@@ -472,27 +489,478 @@
...
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df['labeling_order'] = df.index\n",
+    "dataset_with_results = dataset.df.join(df.set_index('record_id'))\n",
```
Comment (Rensvandeschoot) [inline docs/source/example_api_asreview_file.ipynb:935]:
```suggestion
    "dataset_with_results = dataset.df.join(df.set_index('record_id'))\n",
    "dataset_with_results.to_csv('data_and_state_merged.csv', index=False)",
```

**Review by Rensvandeschoot — CHANGES_REQUESTED** [review]

**Comment by J535D165** [comment]
Nice!

**Pull request merged**